README.md

```diff
@@ -1,8 +1,9 @@
 # Con-PCA API #
 
-[![GitHub Build Status](https://github.com/cisagov/con-pca-api/workflows/build/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/build.yml)
-[![CodeQL](https://github.com/cisagov/con-pca-api/workflows/CodeQL/badge.svg)](https://github.com/cisagov/con-pca-api/actions/workflows/codeql-analysis.yml)
-[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/con-pca-api/badge.svg)](https://snyk.io/test/github/cisagov/con-pca-api)
+[![GitHub Build Status](https://github.com/cisagov/skeleton-docker/workflows/build/badge.svg)](https://github.com/cisagov/skeleton-docker/actions/workflows/build.yml)
+[![CodeQL](https://github.com/cisagov/skeleton-docker/workflows/CodeQL/badge.svg)](https://github.com/cisagov/skeleton-docker/actions/workflows/codeql-analysis.yml)
+[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/skeleton-docker/badge.svg)](https://snyk.io/test/github/cisagov/skeleton-docker)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/skeleton-docker.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/skeleton-docker/context:python)
```

Comment:
Please correct these URLs to point to `cisagov/con-pca-api` instead of `cisagov/skeleton-docker`.

Reply:
@jsf9k resolved with [e87220f](https://github.com/cisagov/con-pca-api/pull/860/commits/e87220f2c52667b8738b039a47608285c99fc713)